modules/iam-read-only-policy/main.tf

```diff
@@ -0,0 +1,90 @@
+resource "aws_iam_policy" "policy" {
+  count = var.create_policy ? 1 : 0
+
+  name        = var.name
+  path        = var.path
+  description = var.description
+
+  policy = data.aws_iam_policy_document.combined.json
+
+  tags = var.tags
+}
+
+locals {
+  console_services = ["resource-groups", "tag", "health"]
```

Comment:
Let's move these from locals to separate variable defaults to allow customizations (if necessary).

Reply:
done